src/code/z_skelanime.c

```diff
@@ -1,5 +1,6 @@
 #include "prevent_bss_reordering.h"
 #include "global.h"
+#include "prevent_bss_reordering.h"
```

Comment:
```suggestion
#include "global.h"
```
Does this work by chance?